protocol/contracts/zaps/ZapBase.sol

```diff
@@ -0,0 +1,103 @@
+pragma solidity ^0.8.4;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+abstract contract ZapBase is Ownable {
+  using SafeERC20 for IERC20;
+
+  bool public paused;
+
+  uint256 internal constant DEADLINE = 0xf000000000000000000000000000000000000000000000000000000000000000;
```

Comment:
I don't know enough about the AMM - but is it ok that this has such a large deadline?

Reply:
modifier in AMM checks for `deadline >= blocktimestamp`